requirements.txt

```diff
@@ -0,0 +1 @@
+mysql_connector_repackaged==0.3.1
```

Comment:
Yes, and I am working on that. Problem is [PyPi](https://pypi.python.org/pypi/mysql-connector) only has up to 2.1.5